packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/extensions/string_extensions.dart

```diff
@@ -0,0 +1,9 @@
+{{> licence.dart }}
+
+import 'dart:convert';
+
+import 'package:crypto/crypto.dart';
+
+extension Sha256 on String {
+  String toSha256() => sha256.convert(utf8.encode(this)).toString();
```

Comment:
Do we still need this class ?

Reply:
Nice catch. This isn't referenced in any part of the project generated with `all_enabled` so we can remove it.

Reply:
Done!